lm_human_preference_details/train_policy_accelerate.py

```diff
@@ -510,6 +514,37 @@ def train(args: Args):
     )
     dataloader = DataLoader(dataset, batch_size=args.ppo.local_batch_size)
     policy, optimizer, dataloader = accelerator.prepare(policy, optimizer, dataloader)
+    if args.deepspeed:
+        import deepspeed
+
+        deepspeed_states = AcceleratorState().deepspeed_plugin
+        deepspeed_states.deepspeed_config['train_micro_batch_size_per_gpu'] = args.ppo.local_micro_batch_size
+        deepspeed_states.deepspeed_config['checkpoint'] = {'use_node_local_storage': True}
```

Comment:
I think this flag is only needed if each node has a separate local filesystem. For the HFC case, you probably don't need it